scripts/verify-pallets-build.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
```

Comment:
I still need to tweak this file but I'll make a separate PR, need to merge these changes asap.